doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -61,7 +65,6 @@ As the next step, the API subscribes to an MQTT topic to start receiving user as
 
 Every time nRF Cloud starts a communication session with a device, it verifies whether the device is uniquely associated with a user.
 If not, the user association procedure is triggered.
-When adding the device to an nRF Cloud account, the user must provide the correct device ID and PIN to nRF Cloud.
```

Comment:
this association section is only applicable for the JITP process.

Reply:
Check it again please.